components/_common_/CloseButton/CloseButton.css

```diff
@@ -0,0 +1,33 @@
+.CloseButton {
+  top: 1rem;
+  right: 1rem;
+  background: var(--slate);
+  border-radius: 50%;
+  border: none;
+  position: absolute;
+}
+
+.CloseButton:disabled {
+  cursor: not-allowed;
+  background-color: var(--slateLight);
```

Comment:
I've removed the CloseButton because it didn't align with the new theme. I'll see about making a new close button with the same API that we can merge into this PR.

Reply:
Close button still seems to be present

Reply:
I merged in an updated CloseButton 👍